ua-snap/fire-tally
# Launch cleanup

Work in progress on a few cleanup items before launch or further work.

 * handle SSL validation failure
 * update libraries to most current versions & remove outdated Dash configuration items
 * Handle env vars more robustly

To test this branch, set these env variables:

```bash
export FLASK_DEBUG=False
export TALLY_DATA_URL=https://fire.ak.blm.gov/content/aicc/Statistics%20Directory/Alaska%20Daily%20Stats%20-%202004%20to%20Present.csv
```

Then run the application.  (It seems to take a weirdly long time to accept connections, just be patient and let it cook for a bit.  Will figure that out soon!)

The app should load and there should be valid data, and you can compare this to the current [beta version](http://fire-tally-dev.us-west-2.elasticbeanstalk.com) to make sure nothing looks too crazy or wrong.
